.github/workflows/opt-out-import-test-integration.yml

```diff
@@ -43,9 +46,46 @@ jobs:
           role-chaining: true
           role-skip-session-tagging: true
       - name: Upload test file to the BFD bucket to trigger lambda function via SNS message
+        id: createfile
         run: |
+          echo "FILENAME=T\#EFT.ON.ACO.NGD1800.DPRF.D$(date +'%y%m%d').T$(date +'%H%M%S')" >> "$GITHUB_OUTPUT"
           aws s3 cp --no-progress ../shared_files/synthetic1800MedicareFiles/test/T\#EFT.ON.ACO.NGD1800.DPRF.D181120.T1000009 \
-            s3://bfd-test-eft/bfdeft01/bcda/in/T.NGD.DPC.RSP.D$(date +'%y%m%d').T$(date +'%H%M%S')1.IN
-
-  # TODO Check bucket for response file
-  # TODO Run another job to check database for update
+            s3://bfd-test-eft/bfdeft01/bcda/in/$FILENAME
+  verify:
+    needs: trigger
+    runs-on: self-hosted
```

Comment:
Add the following here to continue to use node16 instead of node20 for the time being on the runner:

```
    env:
      ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: "true"
```